docs/UserGuide.md

```diff
@@ -295,6 +295,40 @@ Examples:
 * `elist` followed by `edelete 2` deletes the 2nd event in the event list.
 * `efind Beach cleaning` followed by `edelete 1` deletes the 1st event in the results of the `find` command (tentative feature)
 
+### Edit the details of an event: `eedit`
+
+Volunteer coordinators can edit the details of the events.
+
+Format: `eedit EVENT_Id [n/EVENT_NAME] [r/ROLES_NEED]... [sd/START_DATETIME] [ed/END_DATETIME] [l/LOCATION] [dsc/DESCRIPTION] [m/MATERIALS_AND_LOGISTICS_NEEDED]... [b/BUDGET]`
+
+Parameters:
+* n/ - Event name
+* r/ - Roles needed for the event and its quantity
+* sd/ - Start date and time of the event
+* ed/ - End date and time of the event
+* l/ - Location of the event
+* dsc/ - Description of the event
+* m/ - Materials needed for the event and its quantity
+* b/ - Budget for the event
+
+Restrictions:
+* The event index must be valid, i.e. If the list of events displayed is 10 events long, the acceptable values will be from 1-10.
```

Comment:
clarity can be improved, e.g. material and role argument must be a positive integer

Reply:
Adding on, can add one more restriction: `At least one optional field must be provided`

Reply:
Ok fixed